piker/brokers/ib.py

```diff
@@ -1179,3 +1267,63 @@ async def stream_trades(
             continue
 
         yield {'local_trades': (event_name, msg)}
+
+
+@tractor.context
+async def open_symbol_search(
```

Comment:
uses `tractor`'s new context api to do request/response style searches over a stream.